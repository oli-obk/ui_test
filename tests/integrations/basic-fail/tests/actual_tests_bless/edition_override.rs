//@edition:2018
//@check-pass

fn main() {
    match 42 {
        0...10 => {} //~ WARN: deprecated
        _ => {}
    }
}
