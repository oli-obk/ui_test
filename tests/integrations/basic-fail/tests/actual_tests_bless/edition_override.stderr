warning: `...` range patterns are deprecated
 --> tests/actual_tests_bless/edition_override.rs:6:10
  |
6 |         0...10 => {}
  |          ^^^ help: use `..=` for an inclusive range
  |
  = warning: this is accepted in the current edition (Rust 2018) but is a hard error in Rust 2021!
  = note: for more information, see <https://doc.rust-lang.org/nightly/edition-guide/rust-2021/warnings-promoted-to-error.html>
  = note: `#[warn(ellipsis_inclusive_range_patterns)]` on by default

warning: 1 warning emitted

